azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/Checkpoint.java

```diff
@@ -19,7 +19,7 @@
 public class Checkpoint
 {
 	private final String partitionId;
-	private String offset = PartitionReceiver.START_OF_STREAM;
+	private String offset = "-1";
```

Comment:
Can we use EventPositionImpl.START_OF_STREAM instead? There's got to be a better way than sticking an unexplained magic value here.

Reply:
That seems odd too because then we'd have to make `EventPositionImpl.START_OF_STREAM` public. How about having `START_OF_STREAM` (and maybe `END_OF_STREAM`) in `ClientConstants.java`?